Failed to Run Android Instance

I'm trying to get up and running with an Android instance, following the [README](https://github.com/uport-project/uport-mobile/blob/develop/README.md). I was able to get an iOS instance running without issues. However, `react-native run-android` was giving errors.

Steps to reproduce the behavior:
1. Clone the project
2. From that directory, in a terminal window, run `native-react run-android`

**Expected behavior**
I would expect an instance of the application to be installed on the adb connected device.

**Actual behavior**
```
FAILURE: Build failed with an exception.

* What went wrong:
Could not resolve all files for configuration ':react-native-signer-wrapper:debugCompileClasspath'.
> Could not find com.facebook.react:react-native:0.55.3.
  Required by:
      project :react-native-signer-wrapper

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 5s
38 actionable tasks: 1 executed, 37 up-to-date
Could not install the app on the device, read the error above for details.
Make sure you have an Android emulator running or a device connected and have
set up your Android development environment:
https://facebook.github.io/react-native/docs/getting-started.html
```

**Screenshots**
I made sure that I had a device running:
![image](https://user-images.githubusercontent.com/6688146/52965617-7983c080-3362-11e9-855d-3d52cd8a489f.png)

**System:**
 - OS: macOS Mojave
 - Version: 10.14.3

**Smartphone:**
 - Device: Pixel 2XL
 - OS: 5.0 (SDK 21)

**Additional context**
Troubleshooting I've done:
* Using a physical device - this is the actual use-case I'll need this for since I'm working to add bluetooth and location services to add [XYO's BLE SDK](https://developers.xyo.network/docs/en/SDK-Ble-Android/) to pair Proof of Location with Proof of Identity, which was the subject of [the hack](https://github.com/RyanJoseph15/uport-mobile/tree/GeoVerify) my team worked on at [EthDenver](https://www.ethdenver.com/) over the weekend.
* I've also tried running the project from an existing installation of Android Studio, but *"[Open project in Android studio and press play](https://github.com/uport-project/uport-mobile/blob/develop/README.md#run-from-android-studio)"* doesn't seem to to be accurate. When I opened the project, Android Studio did not automatically detect any Run Configurations.
  * A side note on this step, I assumed the correct directory to open was the root directory.
* > Could not find com.facebook.react:react-native:0.55.3.
  * Looking into this from the console output, I wasn't able to resolve the issue after reading through [React-Native](https://facebook.github.io/react-native/docs/getting-started.html)'s documentation.

--- a/.circleci/config.yml
+++ b/.circleci/config.yml
@@ -146,7 +146,7 @@ jobs:
       - run:
           name: xcrun
           command: |
-            cd ~/uport-mobile/ios && xcrun instruments -w "iPhone 8 (11.3) [" || true
+            cd ~/uport-mobile/ios && xcrun instruments -w "iPhone 8 (11.2) [" || true
             pod install --verbose
             npm install
             yarn

--- a/README.md
+++ b/README.md
@@ -48,22 +48,29 @@ $ react-native run-ios
 ### Android setup
 
 This step depends on the [JavaScript environment setup](#javascript-environment-setup).
+
 It's also a good idea to run `yarn` every time you switch branches to sync the `node_modules` folder.
 
-- Easiest way is to install Android Studio. This will manage the SDK for you and makes life easier for non Android devs
+- Easiest way is to install Android Studio. This will manage the android SDK for you and makes life easier for non Android devs
 - Select the custom install option and select options for performance and AVD creation
 - Ensure to install both the minimum SDK 21 and the latest that the current version of React Native supports from the new project settings
 - Since the app uses push messaging to receive signing requests from dApps it needs to be integrated into a Firebase project.
     A `google-services.json` file must be placed in the `android/app/` folder.
-    You can get such a file by creating a firebase project [here](https://console.firebase.google.com)
+    You can get it by creating a project in the [firebase console](https://console.firebase.google.com)
 
 #### Run from Android Studio
-- load the `android` subfolder as a project in Android Studio
+
+- Load the `android` subfolder as a project in Android Studio
+
     **Do not upgrade any dependencies (gradle) if prompted by Android Studio**
+    
 - React native uses a packaging server to enable hot-reloading during debugging.
-    You'll need to run `yarn start` in a terminal window to start one.
+    
+    You'll need to run `yarn start` in a terminal window in the project root folder to start the packager.
     This needs to be done only once for a debugging session and you can use the same server between app restarts.
+    
 - With a packaging server running, you can now run the `app` runConfiguration by pressing the green play button in the tool window.
+
 - It will take a while to start the app for the first time because the packager needs to do it's job, during which you only see a splash-screen.   
 
 #### Run from command line

--- a/README.md
+++ b/README.md
@@ -47,22 +47,33 @@ $ react-native run-ios
 
 ### Android setup
 
+This step depends on the [JavaScript environment setup](#javascript-environment-setup).
+It's also a good idea to run `yarn` every time you switch branches to sync the `node_modules` folder.
+
 - Easiest way is to install Android Studio. This will manage the SDK for you and makes life easier for non Android devs
 - Select the custom install option and select options for performance and AVD creation
-- Ensure to install both the minnimum SDK 21 and the latest that the current version of React Native supports from the new project settings
-
-**Do not upgrade any depencencies (gradle) if prompted by Android Studio**
+- Ensure to install both the minimum SDK 21 and the latest that the current version of React Native supports from the new project settings
+- Since the app uses push messaging to receive signing requests from dApps it needs to be integrated into a Firebase project.
+    A `google-services.json` file must be placed in the `android/app/` folder.
+    You can get such a file by creating a firebase project [here](https://console.firebase.google.com)
 
 #### Run from Android Studio
-Open project in Android studio and press play
+- load the `android` subfolder as a project in Android Studio
+    **Do not upgrade any dependencies (gradle) if prompted by Android Studio**
+- React native uses a packaging server to enable hot-reloading during debugging.
+    You'll need to run `yarn start` in a terminal window to start one.
+    This needs to be done only once for a debugging session and you can use the same server between app restarts.
+- With a packaging server running, you can now run the `app` runConfiguration by pressing the green play button in the tool window.
+- It will take a while to start the app for the first time because the packager needs to do it's job, during which you only see a splash-screen.   
 
 #### Run from command line
-
-After cloning repository
+After cloning repository, running `yarn` and also getting a `google-services.json` file,
+Run this in the project root folder 
 ```bash=
 $ react-native run-android
 ```
-
+The above command also takes care of starting the packaging server.
+It will take a while to start the app for the first time because the packager needs to do it's job, during which you only see a splash-screen.
 
 
 ## StandardJS

